yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtil.scala

```diff
@@ -519,9 +519,9 @@ object YarnSparkHadoopUtil {
       conf: SparkConf,
       numExecutors: Int = DEFAULT_NUMBER_EXECUTORS): Int = {
     if (Utils.isDynamicAllocationEnabled(conf)) {
-      val minNumExecutors = conf.get(DYN_ALLOCATION_MIN_EXECUTORS)
-      val initialNumExecutors = conf.get(DYN_ALLOCATION_INITIAL_EXECUTORS)
-      val maxNumExecutors = conf.get(DYN_ALLOCATION_MAX_EXECUTORS)
+      val minNumExecutors = Utils.getDynamicAllocationMinExecutors(conf)
```

Comment:
Sorry, I thought that DYN_ALLOCATION_MAX_EXECUTORS was a String. Marcelo's comment cleared it up and I've fixed these. Thanks!